<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlcDebug</class>
 <widget class="QWidget" name="PlcDebug">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>619</width>
    <height>255</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PlcDebug</string>
  </property>
  <widget class="QLineEdit" name="lineEdit_RAddr">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>40</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_RVal">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>40</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>40</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>PLC地址</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>40</y>
     <width>54</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>读取数值</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_R">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>40</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>MD</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>MW</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>QW</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>MX</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>QX</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>IX</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>150</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>PLC地址</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_W">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>150</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>MD</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>MW</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>QW</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>MX</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>QX</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>IX</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="lineEdit_WVal">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>150</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_WAddr">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>150</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>150</y>
     <width>54</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>写入数值</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_R">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>90</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>读取</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_W">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>200</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>写入 </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_RPos">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>40</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_WPos">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>150</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
